GVRf/Framework/framework/src/main/java/org/gearvrf/GVRSceneObject.java

```diff
@@ -1219,6 +1219,13 @@ public final BoundingVolume expandBoundingVolume(
     public final BoundingVolume expandBoundingVolume(final Vector3f center, final float radius) {
         return expandBoundingVolume(center.x, center.y, center.z, radius);
     }
+    
+    /*
+     * Check if the object is culled in the frame or not
```

Comment:
I'd say when the need arises we expose it. Otherwise we just confuse people. Unless you can give a specific example where application developers can find it useful? And how to correctly use it since this could be called by any thread.